admin/roles/class-abstract-role-manager.php

```diff
@@ -106,9 +106,9 @@ protected function capability_exists( WP_Role $role, $capability ) {
 	 * @param string $role         The role to check against.
 	 * @param array  $capabilities The capabilities that should be set.
 	 *
-	 * @return array Capabilties that can be safely set.
+	 * @return array Capabilities that can be safely set.
 	 */
-	protected function filter_existing_capabilties( $role, array $capabilities ) {
+	protected function filter_existing_capabilities( $role, array $capabilities ) {
```

Comment:
Changing a method name requires a deprecation of the old method name as it is a breaking change.

Reply:
I see.
I leave it to your engineers.
This PR only show the problem.